### PR TITLE
Adding the links below the graph for Tags page

### LINKED ISSUE #5090: Implement new design for Tags page (needs breaking up)
Building on #4922, we should work to rebuild the page at https://publiclab.org/tags following this design:

Screen Shot 2019-03-15 at 12 45 20 PM

There's discussion about this here: https://publiclab.org/notes/edie_blues/01-09-2019/the-tags-page-mockup-let-me-know-what-you-think#c22154

The current template is here: https://github.com/publiclab/plots2/tree/master/app/views/tag/index.html.erb

Steps that could be broken out include:

* [x] making it 2-column #5100
* [x] adding https://publiclab.org/stats/graph in an iframe to the 2nd column
* [x] adding the links below the graph
* [x] Redesigning the search bar (https://github.com/publiclab/plots2/pull/5121)
* [ ] redesigning the table to use a Card ui instead (starting with panels, here: https://getbootstrap.com/docs/3.3/components/#panels) in a template partial

I'd love help with this one!

### PR DESCRIPTION
**Pull request by gautamig54**
Fixes #5090  (<=== Add issue number here)
Fixing the 3rd task of adding links below the graph for the new design of the tags page.

Screen Shot 2019-03-21 at 2 43 10 AM

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by CleverFool77** [comment]
The image looks so close to the aim.. 😍

**Comment by gautamig54** [comment]
@jywarren Kindly look into the PR and merge if all seem fine. Thanks!

File: app/views/tag/index.html.erb [hunk 1]
```diff
@@ -58,5 +58,12 @@
...
 <div class="col-md-6">
 <br><br>
-<iframe src="stats/graph.html?limit=50" height="500" width="100%" frameborder="0" ></iframe>
+<iframe id="myImage" src="stats/graph.html?limit=50" height="500" width="100%" frameborder="0" ></iframe>
+<h6 class="text-right" style="color:#ccc;">Tags most commonly used, linked by how often they are used together.<br>
+  View <a style="cursor: pointer;" onclick="document.getElementById('myImage').src='stats/graph.html?limit=50'"><u>50</u></a> |
+  <a style="cursor: pointer;" onclick="document.getElementById('myImage').src='stats/graph.html?limit=100'"><u>100</u></a> |
+  <a style="cursor: pointer;" onclick="document.getElementById('myImage').src='stats/graph.html?limit=250'"><u>250</u></a> |
+  <a style="cursor: pointer;" onclick="document.getElementById('myImage').src='stats/graph.html?limit=500'"><u>500</u></a>
+</h6>
```
Comment (gauravano) [inline app/views/tag/index.html.erb]: Please indent the code.

**Comment by gautamig54** [comment]
@gauravano sure i'll do it. Thanks!

File: app/views/tag/index.html.erb (hunk 1 above)
Comment (jywarren) [reply app/views/tag/index.html.erb]: wow this is such a clever solution! Thank you!

**Comment by jywarren** [comment]
If it's possible to highlight the link and you can share a screenshot of that working, we're happy to merge this, thanks!

File: app/views/tag/index.html.erb (hunk 1 above)
Comment (gautamig54) [reply app/views/tag/index.html.erb]: Thank you @jywarren @gauravano!

**Comment by gautamig54** [comment]
Okay @jywarren. I'll highlight the active link and make a PR. Should I make a different PR or push commit here?

**Comment by gautamig54** [comment]
@jywarren @gauravano I have implemented highlighting the active link and pushed commit to this PR. Kindly look into it. Thanks!

Screen Shot 2019-03-22 at 12 42 17 AM

Screen Shot 2019-03-22 at 12 42 24 AM

**Comment by jywarren** [comment]
This is so nice!!! I'm merging now. Great work!

**Comment by gautamig54** [comment]
Thanks @jywarren

**Comment by jywarren** [comment]
Ok,  done! Thanks!

**Pull request merged**